wolfcrypt/src/asn.c

```diff
@@ -9697,7 +9703,9 @@ static int DecodeBasicOcspResponse(byte* source, word32* ioIndex,
             return ASN_PARSE_E;
 
         InitDecodedCert(&cert, resp->cert, resp->certSz, heap);
-        ret = ParseCertRelative(&cert, CERT_TYPE, VERIFY, cm);
+        /* Don't verify if we don't have access to Cert Manager. */
+        ret = ParseCertRelative(&cert, CERT_TYPE,
```

Comment:
OcspResponseDecode is the public API that is called by the OpenSSL compatibility code.
Do I add a parameter or add a new API like: OcspResponseDecode_ex?